README.md

```diff
@@ -21,7 +21,7 @@ as Java system properties (-D flags) or as environment variables. This is an exa
 
 ```
 java -javaagent:/path/to/opentelemetry-auto-<version>.jar \
-     -Dota.exporter=jaeger \
+     -Dota.exporter.jar=exporter-adapters/dummy-exporter-adapter/build/libs/dummy-exporter-adapter-0.1.2-SNAPSHOT.jar \
```

Comment:
i think keep the example using jaeger?

Reply:
I do, but the ota.exporter flag is gone and you have to point to the jar, since we no longer bundle the jaeger exporter.